tests/playwright-test/fixture-errors.spec.ts

```diff
@@ -367,7 +367,7 @@ test('should print nice error message for problematic fixtures', async ({ runInl
   });
   expect(result.exitCode).toBe(1);
   expect(result.output).toContain('oh my!');
-  expect(result.output).toContain('x.spec.ts:5:29');
+  expect(result.output).toContain('x.spec.ts:6:49');
```

Comment:
the new column looks right but why did this change? Its hard for me to verify the line number.

Reply:
It was using location of `.extend()` instead of location of `new Error()`.